modules/quota-increase/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = ">= 0.12.26"
+}
+
+resource "aws_servicequotas_service_quota" "increase_quotas" {
+  for_each = var.resources_to_increase
+
+  quota_code = local.codes[each.key].quota_code
+  service_code = local.codes[each.key].service_code
+  value = each.value
+}
+
+locals {
+  codes = {
+    nacl_rules = {
+      quota_code = "L-2AEEBF1A"
```

Comment:
Do these stay consistent across all regions?

Reply:
Oh, actually, can these codes be looked up using https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/servicequotas_service?

Reply:
> Oh, actually, can these codes be looked up using https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/servicequotas_service?

No, what this one returns is the high level, like  `vpc`, but not the code for nacl rules, for example.

I could not find on the docs if this code is the same across regions BUT since it's fine to request over and over the same limit, I'll add a test to check if after apply the `quota_name` and `service_name` are correct.

When I worked on the python script, the same code worked for all regions, I don't know if this is the case for all resources.

Reply:
The test is checking if the request was made to the correct resources! If it runs in a region that has a different code it'll break and we'll know!

Reply:
Roger!